ci/cached-builds/gha_lvm_overlay.sh

```
@@ -0,0 +1,78 @@
+/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)